### ISSUE TITLE
Partial type imports does not work

### ISSUE THREAD

**Issue by ottoville** [pahen/madge#330]
When typescript file has import with this style

`import { getResponse, type APIResponseType} from "./api";
`
Magde will not resolve any dependencies, running with --debug will print this error

```
  tree error getting dependencies: ',' expected. +30ms
  tree TSError: ',' expected.
```

**Comment by fdc-viktor-luft** [comment]
@kamiazya might this be fixed with V6 release?

**Comment by PabloLION** [comment]
I had the same issue and my work around was to split the line like this
```ts
import { getResponse } from "./api"; 
import type { APIResponseType } from "./api"; 
```

**Comment by kamiazya** [comment]
@fdc-viktor-luft cc @PabloLION 
Thank you!

madge v6 will update detective-typescript to 9.0.0, and it's depends on typescript@^4.5.5.

`Type-Only Imports and Export` syntax is implemented in [typescript@3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export).

So this issue will be resolved.

**Comment by kamiazya** [comment]
resolved in #321
